ome/omero-figure
# Points in shape editor

NB: this is on top of Vite PR #477 - needs to be merged first.

As discussed at https://github.com/ome/omero-iviewer/pull/465#issuecomment-2066771147, need to support Points in OMERO.figure.

Points were already supported in `Figure_To_Pdf.py`.

To test:

 - Add Points from OMERO
 - Draw, update (drag, change colour), copy & paste Points
 - Save Points to the Figure - open Figure to check they are correct
 - Export Figure to PDF and to TIFF
